plugins/inputs/cpu/cpu.go

```diff
@@ -97,7 +97,7 @@ func (s *CPUStats) Gather(acc telegraf.Accumulator) error {
 		totalDelta := total - lastTotal
 
 		if totalDelta < 0 {
-			err = fmt.Errorf("Error: current total CPU time is less than previous total CPU time")
+			err = fmt.Errorf("error: current total CPU time is less than previous total CPU time")
```

Comment:
`error:` is redundant and should be removed IMO.

Reply:
Removed